### ISSUE TITLE
Allow scalar outputs

### ISSUE THREAD

**Issue by blazekadam** [iterait/emloop-tensorflow#40]
We do not support scalar outputs... wtf?

**Comment by petrbel** [comment]
AFAIK at least batch-dim must be provided. We run into this when implementing tf-liter.

**Comment by blazekadam** [comment]
Yes, that is the case. Even worse all the outputs have to have the very same batch dimension. I remember the dark rainy day when I implemented it...
